app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Admin
+  class TicketScanningsController < ApplicationController
+    before_action :authenticate_user!
+    authorize_resource
+
+    def create
+      @physical_ticket = PhysicalTicket.find_by_token(params[:id])
```

Comment:
Load resource is not working here. I am not sure if load_resource work for create action.